controller/src/main/java/gov/va/vro/controller/VroController.java

```diff
@@ -59,7 +56,7 @@ public ResponseEntity<GeneratePdfResponse> generatePdf(GeneratePdfRequest reques
       GeneratePdfPayload model = generatePdfRequestMapper.toModel(request);
       log.info(model.toString());
       String response = camelEntrance.generatePdf(model);
-      log.info(response.toString());
+      log.info(response);
```

Comment:
Do not log whole responses please. We still do not want PHI in our logs.

Reply:
I didn't! I was only simplifying the expression based on the IDE recommendation. But I will remove this